src/pipelines/html.rs

```diff
@@ -148,17 +148,24 @@ impl HtmlPipeline {
             r#"only one <link data-trunk rel="rust" data-type="main" .../> may be specified"#
         );
         if rust_app_nodes == 0 {
-            if let Some(app) = RustApp::new_default(
-                self.cfg.clone(),
-                self.target_html_dir.clone(),
-                self.ignore_chan.clone(),
-            )
-            .await?
-            {
-                assets.push(TrunkAsset::RustApp(app));
-            } else {
-                tracing::warn!("no rust project found")
-            };
+            match self.cfg.build_target {
```

Comment:
I don't think that not building the project is the wrong approach here. I guess that it would need to be added here: https://github.com/trunk-rs/trunk/blob/d61478448bd093449708b63dcb3e7baf286a191c/src/pipelines/rust/mod.rs#L352-L357